pkg/v1/tkg/client/get_cluster_pinniped_info.go

```diff
@@ -52,6 +52,7 @@ func (c *TkgClient) GetClusterPinnipedInfo(options GetClusterPinnipedInfoOptions
 		return nil, errors.Wrap(err, "error determining 'Tanzu Kubernetes Cluster service for vSphere' management cluster")
 	}
 	if isPacific {
+		// XXX: Override this path? (is there existing work to remove these pacific specific conditionals?)
```

Comment:
Question: `GetClusterPinnipedInfo` method would be used to get the kubeconfig of both management cluster and workload cluster using the command `tanzu mc kubeconfig get` and `tanzu cluster kubeconfig get <workload_cluster_name>`.  So would like to know what is the expected behavior after the management cluster(supervisor) is added to tanzu.

Reply:
Hmm. I think we would want `tanzu cluster kubeconfig get` to work for sure. I think making it work could be a followup patch?

I'm leaning towards having `mc kubeconfig get` continue to error out given that-

 1) there isn't necessarily an admin kubeconfig available on vSphere with Tanzu
2) anyone can run tanzu login against the management cluster to get the same kubeconfig that mc kubeconfig get would have yielded.

Thoughts?

Reply:
> I think we would want tanzu cluster kubeconfig get to work for sure. I think making it work could be a followup patch?

Sounds good to me. Just want to point out `kubeconfig` subcommands for both `tanzu mc` and `tanzu cluster` commands would need to be addressed. 

> 1. there isn't necessarily an admin kubeconfig available on vSphere with Tanzu
> 2.anyone can run tanzu login against the management cluster to get the same kubeconfig that mc kubeconfig get would have yielded.

We do have options for `mc kubeconfig get` 
1. `tanzu mc kubeconfig get --admin` to get admin kubeconfig
2.  `tanzu mc kubeconfig get ` to get pinniped based kubeconfig 
So I believe you are saying for 1) we can continue to error out. But for 2)  we can return the same kubeconfig we get from login API. But I would let IAM team(@ankeesler, @sabbey37, @benjaminapetersen ) to provide the guidance here cc:  @vuil . We can create an Issue for this and track the discussion there.

Reply:
Yeah 2) could swing either way from my perspective, but I don't think it's a significant amount of work to go either way.